sdk/formrecognizer/ai-form-recognizer/samples/v5/javascript/README.md

```diff
@@ -0,0 +1,88 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-form-recognizer
```

Comment:
```suggestion
  - azure-form-recognizer
  - azure-ai-document-intelligence
```
?

Reply:
These are special and curated by the docs team. Going to wait on updating this one.